### ISSUE TITLE
[dexter-vs] System.ArgumentException when clicking on Properties

### ISSUE THREAD

**Issue by mdopark** [Samsung/Dexter#153]
System.ArgumentException occurs when doublic-clicking on Properties node in Solution explorer.
Below is the details of the exception.
```System.ArgumentException이(가) 발생했습니다.
  HResult=0x80070057
  메시지=The parameter is incorrect. . (예외가 발생한 HRESULT: 0x80070057 (E_INVALIDARG))
  소스=
  StackTrace:
   / EnvDTE._DTE.get_ActiveDocument()
   / Dexter.Common.Config.Providers.FileInfoProvider.Load() 파일 D:\dexter\github\vs2017\Dexter\project\dexter-vs\Dexter\Common\Core\Config\Providers\FileInfoProvider.cs:줄 34
   / Dexter.Common.Config.Providers.ConfigurationProvider.LoadProjectInfo() 파일 D:\dexter\github\vs2017\Dexter\project\dexter-vs\Dexter\Common\Core\Config\Providers\ConfigurationProvider.cs:줄 55
   / Dexter.UI.Analysis.DexterAnalysisCommand.Refresh() 파일 D:\dexter\github\vs2017\Dexter\project\dexter-vs\Dexter\Core\UI\Analysis\DexterAnalysisCommand.cs:줄 177
   / Dexter.UI.Analysis.DexterFileAnalysisCommand.OnDocumentWindowActivated(Window gotFocus, Window lostFocus) 파일 D:\dexter\github\vs2017\Dexter\project\dexter-vs\Dexter\Core\UI\Analysis\DexterFileAnalysisCommand.cs:줄 48
   / Microsoft.VisualStudio.Platform.WindowManagement.DTE.WindowEvents.InternalFireActivated(Window gotFocus, Window lostFocus)```

**Comment by KarolAntczak** [comment]
I will check this out.

**Comment by mdopark** [comment]
It was resolved by #154.
